articles/videos/get-started/01-architecture-your-tenant.md

```diff
@@ -0,0 +1,159 @@
+---
+description: What an Auth0 tenant is and how to configure it in the Auth0 Dashboard.
+classes: video-page
+---
+# Architecture: Your Tenant
+
+Learn what an Auth0 tenant is and how to configure it in the Auth0 Dashboard. Understand why you may want more than one tenant if you have different user communities, and also how you can use more than one tenant to support your Software Development Life Cycle. Learn the importance of tenant naming, use of custom domains, provisioning additional tenant administrators, and how to associate tenants with your Auth0 account.
```

Comment:
For consistency with other pages, add "(SDLC)" after "Software Development Life Cycle".

Last sentence is awkward. Missing verbs or an "and" somewhere?

Reply:
Fixed